Makefile

```diff
@@ -7,6 +7,11 @@ PASSWORD_ENCRYPTED=
 ## Select bash for commands
 SHELL := /bin/bash
 
+## Path to vendor linux patch folder
+ifeq ($(MAKECMDGOALS),cavm-all)
+export LINUX_PATCH=cavm-as7512
+endif
```

Comment:
suggest drop this for now. Let's first agree on how the sonic kernel is going to be build in the other repo